README.md

```diff
@@ -6,6 +6,41 @@
 
 Feign is a Java to HTTP client binder inspired by [Retrofit](https://github.com/square/retrofit), [JAXRS-2.0](https://jax-rs-spec.java.net/nonav/2.0/apidocs/index.html), and [WebSocket](http://www.oracle.com/technetwork/articles/java/jsr356-1937161.html).  Feign's first goal was reducing the complexity of binding [Denominator](https://github.com/Netflix/Denominator) uniformly to HTTP APIs regardless of [ReSTfulness](http://www.slideshare.net/adrianfcole/99problems).
 
+---
+# Roadmap
+## Feign 11 and beyond 
+Making _API_ clients easier
+
+Short Term - What we're working on now. ⏰ 
+---
+* Response Caching
+  * Support caching of api responses.  Allow for user's to define under what conditions a response is eligible for caching and what type of caching mechanism should be used.
+  * Support in-memory caching and external cache implementations (EhCache, Google, Spring, etc...)
+* Complete URI Template expression support
+  * Support [level 1 through level 4](https://tools.ietf.org/html/rfc6570#section-1.2) URI template expressions.
+  * Use [URI Templates TCK](https://github.com/uri-templates/uritemplate-test) to verify compliance.
+* `Logger` API refactor
+  * Refactor the `Logger` API to adhere closer to frameworks like SLF4J providing a common mental model for logging within Feign.  This model will be used by Feign itself throughout and provide clearer direction on how the `Logger` will be used.
+* `Retry` API refactor
+  * Refactor the `Retry` API to support user-supplied conditions and better control over back-off policies. 
+
+Medium Term - What's up next. ⏲ 
+---
+* Metric API
+  * Provide a first-class Metrics API that user's can tap into to gain insight into the request/response lifecycle.  Possibly provide better [OpenTracing](https://opentracing.io/) support.
```

Comment:
I was more thinking about timers and meters, not tracing, which is a nice to have too

https://metrics.dropwizard.io/4.0.0/